learn/dapps-list/dapp-directory.md

```diff
@@ -0,0 +1,299 @@
+|          `projectCreationDate`          |                          int                          |                                                                                                                             The date the project was created. Used for sorting purposes in the DApp Directory                                                                                                                              |
+
+??? code "Example project data file"
+
+    ```json
+    --8<-- 'code/learn/dapps-list/dapp-directory/project-data-file.json'
+    ```
+
+#### Category and Tags {: #category-and-tags }
+
+The currently supported values for `category` and `tags` are:
+
+--8<-- 'text/learn/dapps-list/dapp-directory/category-tags.md'
+
+#### URLs {: #urls }
+
+The `urls` property name/value pairs are used so a project can provide links to their website, socials, etc.
+
+The format of the property name/value pairs should follow the JSON standard, for example:
+
+```json
+--8<-- 'code/learn/dapps-list/dapp-directory/urls.json'
+```
+
+The following table lists the supported `urls` properties:
+
+| Property Name |                                                                       Description                                                                        |
+|:-------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|   `website`   |                                            The main website for the project. E.g., https://moonbeam.network/                                             |
+|     `try`     | URL a user should visit if they want to try out the dApp. Typically, this page will have a link to launch the dApp. E.g., https://moonbeam.network/ |
+|   `twitter`   |                                         The project's Twitter profile. E.g., https://twitter.com/MoonbeamNetwork                                         |
+|   `medium`    |                                           The project's Medium site. E.g., https://medium.com/moonbeam-network                                           |
+|  `telegram`   |                                               The project's Telegram. E.g., https://t.me/Moonbeam_Official                                               |
+|   `github`    |                                  The project's GitHub repository. E.g., https://github.com/moonbeam-foundation/moonbeam                                  |
+|   `discord`   |                                             The project's Discord. E.g., https://discord.com/invite/PfpUATX                                              |
```

Comment:
I don't want to make more work for you but if it's not too much trouble, make it makes sense to make an example column and put the example in that column formatted with `` ?

Reply:
Yeah, I didn't initially do this here because there are other tables where we use E.g., whatever example, but I think it does look better if it's done here!